charts/osm/README.md

```diff
@@ -93,13 +93,13 @@ The following table lists the configurable parameters of the osm chart and their
 | OpenServiceMesh.image.registry | string | `"openservicemesh"` | `osm-controller` image registry |
 | OpenServiceMesh.image.tag | string | `"v0.7.0"` | `osm-controller` image tag |
 | OpenServiceMesh.imagePullSecrets | list | `[]` | `osm-controller` image pull secret |
-| OpenServiceMesh.injector | object | `{"replicaCount":1,"resource":{"limits":{"cpu":"0.5","memory":"32M"},"requests":{"cpu":"0.3","memory":"32M"}}}` | Sidecar injector configuration |
+| OpenServiceMesh.injector | object | `{"replicaCount":1,"resource":{"limits":{"cpu":"0.5","memory":"64M"},"requests":{"cpu":"0.3","memory":"64M"}}}` | Sidecar injector configuration |
 | OpenServiceMesh.meshName | string | `"osm"` | Name for the new control plane instance |
 | OpenServiceMesh.osmNamespace | string | `""` | Optional parameter. If not specified, the release namespace is used to deploy the osm components. |
 | OpenServiceMesh.osmcontroller.resource.limits.cpu | string | `"1.5"` |  |
 | OpenServiceMesh.osmcontroller.resource.limits.memory | string | `"512M"` |  |
 | OpenServiceMesh.osmcontroller.resource.requests.cpu | string | `"0.5"` |  |
-| OpenServiceMesh.osmcontroller.resource.requests.memory | string | `"32M"` |  |
+| OpenServiceMesh.osmcontroller.resource.requests.memory | string | `"128M"` |  |
```

Comment:
wait, did you see this affect anyhow?

Reply:
I didn't see this affected, but I noticed in the [scale doc](https://github.com/openservicemesh/osm/blob/3f0ea1977fb17ff41cb691a6dcbd81d25d4005b0/docs/content/docs/scale.md) that initial memory use was 45 MB so decided to bump this as well. What do you think?

Reply:
if nothing was observed, I'd think it's totally fine